puma_motor_driver/CMakeLists.txt

```diff
@@ -42,13 +42,12 @@ install(TARGETS multi_puma_node
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
-  # the following line skips the linter which checks for copyrights
-  # comment the line when a copyright and license is added to all source files
-  set(ament_cmake_copyright_FOUND TRUE)
-  # the following line skips cpplint (only works in a git repo)
-  # comment the line when this package is in a git repo and when
-  # a copyright and license is added to all source files
-  set(ament_cmake_cpplint_FOUND TRUE)
+  list(APPEND AMENT_LINT_AUTO_EXCLUDE
+    ament_cmake_copyright
```

Comment:
Does this pass if enabled?

Reply:
No:
```
test 1
    Start 1: copyright

1: Test command: /usr/bin/python3 "-u" "/opt/ros/jazzy/share/ament_cmake_test/cmake/run_test.py" "/home/civerachb/Code/clearpath_common_ws/build/puma_motor_driver/test_results/puma_motor_driver/copyright.xunit.xml" "--package-name" "puma_motor_driver" "--output-file" "/home/civerachb/Code/clearpath_common_ws/build/puma_motor_driver/ament_copyright/copyright.txt" "--command" "/opt/ros/jazzy/bin/ament_copyright" "--xunit-file" "/home/civerachb/Code/clearpath_common_ws/build/puma_motor_driver/test_results/puma_motor_driver/copyright.xunit.xml"
1: Working Directory: /home/civerachb/Code/clearpath_common_ws/src/clearpath_motor_drivers/puma_motor_driver
1: Test timeout computed to be: 200
1: -- run_test.py: invoking following command in '/home/civerachb/Code/clearpath_common_ws/src/clearpath_motor_drivers/puma_motor_driver':
1:  - /opt/ros/jazzy/bin/ament_copyright --xunit-file /home/civerachb/Code/clearpath_common_ws/build/puma_motor_driver/test_results/puma_motor_driver/copyright.xunit.xml
1: include/puma_motor_driver/can_proto.hpp: copyright=Texas Instruments Incorporated.  All rights reserved. (2008-2013), license=<unknown>
1: include/puma_motor_driver/driver.hpp: copyright=Clearpath Robotics, Inc., All rights reserved. (2015), license=<unknown>
1: include/puma_motor_driver/multi_puma_node.hpp: copyright=Clearpath Robotics, Inc., All rights reserved. (2024), license=<unknown>
1: src/driver.cpp: copyright=Clearpath Robotics, Inc., All rights reserved. (2015), license=<unknown>
1: src/multi_puma_node.cpp: copyright=Clearpath Robotics, Inc., All rights reserved. (2024), license=<unknown>
1: 5 errors, checked 5 files
1: -- run_test.py: return code 1
1: -- run_test.py: verify result file '/home/civerachb/Code/clearpath_common_ws/build/puma_motor_driver/test_results/puma_motor_driver/copyright.xunit.xml'
1/4 Test #1: copyright ........................***Failed    0.59 sec
```

Reply:
Even if we added a "clearpath" copyright type the can_proto file would fail. There might be a way to set the test to pass if any copyright is found.

Reply:
It doesn't look like you can... So the best option would be to copy what otto did and also exclude the TI file.